qemu/tests/hotplug_virtio_mem.py

```diff
@@ -0,0 +1,55 @@
+import time
+
+from virttest import error_context
+
+from virttest.utils_misc import normalize_data_size
+from virttest.utils_test.qemu import MemoryHotplugTest
+from provider import virtio_mem_utils
+
+
+@error_context.context_aware
+def run(test, params, env):
+    """
+    Virtio-mem hotplug test
+    1) Boot guest
+    2) Hotplug virtio-mem device
+    3) Check virtio-mem device
+    4) Resize virtio-mem device twice
+    5) Check virtio-mem device
+
+    :param test: QEMU test object
+    :param params: Dictionary with the test parameters
+    :param env: Dictionary with test environment
+    """
+
+    timeout = params.get_numeric("login_timeout", 240)
+    threshold = params.get_numeric("threshold", target_type=float)
+    vm = env.get_vm(params["main_vm"])
+    vm.verify_alive()
+    session = vm.wait_for_login(timeout=timeout)
+    virtio_mem_model = params.get("virtio_mem_model", "virtio-mem-pci")
+
+    target_mem = params.get("target_mems")
+    hotplug_test = MemoryHotplugTest(test, params, env)
+    hotplug_test.hotplug_memory(vm, target_mem)
+
+    for i, vmem_dev in enumerate(vm.devices.get_by_params({'driver': virtio_mem_model})):
```

Comment:
IMO, it is more better to use the id of the new pluged virtio mem device `virtio_mem-$target_mem` and verify it.
The definition for `virtio_mem_modle` is not related with device hotplug,  avocado-vt know which model to use as long as `vm_memdev_model_vmem0 = "virtio-mem"`.

Reply:
Changed to use the `"virtio_mem-%s" % target_mem` instead of iterating the vm devices based on the virtio_mem_model, thanks!